_layouts/partners/impact-story.html

```diff
@@ -18,6 +18,13 @@ <h1 class="partners-title">{{ page.title }}</h1>
                     <div class="partners-body padding-top-2">{{ page.challenge | markdownify }}</div>
                     <div class="partners-body padding-top-2">{{ page.solution | markdownify }}</div>
                     <div class="partners-body padding-top-2">{{ page.impact | markdownify }}</div>
+                    {% if page.callout_tips %}                    
+                        <div class="grid-row grid-gap-6">
+                            <div class="tablet:grid-col-12 partners-subtitle partners-blue-box">
+                              <div>{{ page.callout_tips | markdownify }}</div>
```

Comment:
Is the empty `<div>` wrapper necessary?

```suggestion
                              {{ page.callout_tips | markdownify }}
```

Reply:
@aduth This is a direct mirror of the following code:

`identity-site/_layouts/partners/get-started.html`

```
<div class="tablet:grid-col-6 partners-subtitle partners-blue-box">
    <div>{{ page.iaa_tips | markdownify }}</div>
</div>
```

The code is seemingly unnecessary but benign. We should probably remove it from both locations.

Reply:
Removed the div in multiple locations.